vk_video_encoder/libs/VkVideoEncoder/VkEncoderConfigH264.cpp

```diff
@@ -478,12 +478,6 @@ bool EncoderConfigH264::GetRateControlParameters(VkVideoEncodeRateControlInfoKHR
     pRateControlLayersInfo->frameRateNumerator = frameRateNumerator;
     pRateControlLayersInfo->frameRateDenominator = frameRateDenominator;
 
-    if (rateControlMode == VK_VIDEO_ENCODE_RATE_CONTROL_MODE_DEFAULT_KHR) {
-        pRateControlInfo->rateControlMode = VK_VIDEO_ENCODE_RATE_CONTROL_MODE_VBR_BIT_KHR;
-    } else {
-        pRateControlInfo->rateControlMode = rateControlMode;
```

Comment:
So, with this change, we will not support anything but disabled RC?
What happened with:
pRateControlInfo->rateControlMode = rateControlMode;

Reply:
I have been a bit blunt here :)